We have to use fixed s2k-count number otherwise the gpg2 would
want to consult gpg-agent which is not yet installed in the mock
chroot.
Index: gnupg-2.0.17/tests/openpgp/conventional-mdc.test
===================================================================
--- gnupg-2.0.17.orig/tests/openpgp/conventional-mdc.test
+++ gnupg-2.0.17/tests/openpgp/conventional-mdc.test
@@ -21,9 +21,9 @@ for ciph in `all_cipher_algos`; do
         dd if=data-80000 of=z bs=1 count=$i 2>/dev/null
     fi
     echo "Hier spricht HAL" | $GPG --passphrase-fd 0 \
-	--force-mdc --cipher $ciph -c -o x --yes z
+	--force-mdc --cipher $ciph -c -o x --yes --s2k-count 65536 z
     echo "Hier spricht HAL" | $GPG --passphrase-fd 0 \
-	-o y --yes x
+	-o y --yes --s2k-count 96 x
     cmp z y || error "$ciph/$i: mismatch"
   done
 done
Index: gnupg-2.0.17/tests/openpgp/conventional.test
===================================================================
--- gnupg-2.0.17.orig/tests/openpgp/conventional.test
+++ gnupg-2.0.17/tests/openpgp/conventional.test
@@ -12,8 +12,8 @@
 
 #info Checking conventional encryption
 for i in plain-2 data-32000 ; do
-    echo "Hier spricht HAL" | $GPG --passphrase-fd 0 -c -o x --yes $i
-    echo "Hier spricht HAL" | $GPG --passphrase-fd 0    -o y --yes x
+    echo "Hier spricht HAL" | $GPG --passphrase-fd 0 -c -o x --yes --s2k-count 65536 $i
+    echo "Hier spricht HAL" | $GPG --passphrase-fd 0    -o y --yes --s2k-count 65536 x
     cmp $i y || error "$i: mismatch"
 done
 
@@ -21,8 +21,8 @@ for a in `all_cipher_algos`; do
     echo_n "$a "
     for i in plain-1 data-80000 ; do
       echo "Hier spricht HAL" | $GPG --passphrase-fd 0 \
-					  --cipher-algo $a -c -o x --yes $i
-      echo "Hier spricht HAL" | $GPG --passphrase-fd 0 -o y --yes x
+					  --cipher-algo $a -c -o x --yes --s2k-count 65536 $i
+      echo "Hier spricht HAL" | $GPG --passphrase-fd 0 -o y --yes --s2k-count 65536 x
       cmp $i y || error "$i: ($a) mismatch"
     done
 done
Index: gnupg-2.0.17/tests/openpgp/genkey1024.test
===================================================================
--- gnupg-2.0.17.orig/tests/openpgp/genkey1024.test
+++ gnupg-2.0.17/tests/openpgp/genkey1024.test
@@ -10,7 +10,7 @@
 
 . $srcdir/defs.inc || exit 3
 
-$GPG --quiet --batch --debug-quick-random --gen-key <<EOF
+$GPG --quiet --batch --debug-quick-random --s2k-count 65536 --gen-key <<EOF
 Key-Type: DSA
 Key-Length: 1024
 Subkey-Type: ELG
@@ -24,7 +24,7 @@ Passphrase: abc
 EOF
 
 if have_pubkey_algo "RSA"; then
-$GPG --quiet --batch --debug-quick-random --gen-key <<EOF
+$GPG --quiet --batch --debug-quick-random --s2k-count 65536 --gen-key <<EOF
 Key-Type: RSA
 Key-Length: 1024
 Key-Usage: sign,encrypt
